spring-integration-core/src/main/java/org/springframework/integration/support/management/IntegrationManagement.java

```diff
@@ -19,13 +19,19 @@
 import org.springframework.jmx.export.annotation.ManagedOperation;
 
 /**
- * Base interface containing methods to control basic statistics gathering.
+ * Base interface for Integration managed components.
  *
  * @author Gary Russell
  * @since 4.2
  *
  */
-public interface CountsEnablement {
+public interface IntegrationManagement {
+
+	@ManagedAttribute(description = "Use to disable all debug logging on the happy path")
```

Comment:
Looks like not technical sentence, but I can live with that if you insist that it isn't important and we go ahead with other work :smile: